### PR TITLE
Node Best Practices: Project Structure

### PR DESCRIPTION
**Pull request by milesflo**

### CODE REVIEW

**Comment by enki-hq** [comment]
bad indentation of a sequence entry at line 3, column 34:
      - [Project Structure Practices](https://github.com/i0natan/node ... 
                                     ^

**Comment by enki-hq** [comment]
bad indentation of a sequence entry at line 3, column 34:
      - [Project Structure Practices](https://github.com/i0natan/node ... 
                                     ^

File: JavaScript/Node/best-practices/project-structure-continued.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+links:
+
+  - >-
+    [Project Structure Practices](https://github.com/i0natan/nodebestpractices#1-project-structure-practices)
+
+---
+## Content
+
+On top of abiding by Separation of Concerns, there are some additional NodeJS-specific best practices that we should address. These are covered with [Express](http://expressjs.com/) in mind, but you can substitute it with any similar HTTP server package.
```
Comment (lizTheDeveloper) [inline JavaScript/Node/best-practices/project-structure-continued.md]: Remove this paragraph and then below, change in general to "with regards to [Express](http://expressjs.com/), we should..."

**Comment by milesflo** [comment]
Looks good

File: JavaScript/Node/best-practices/project-struture.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Instead, I would want to divide concerns for my API and my web client into different subdirectories and import them into my main `app.js` script with `require()` statements. Organizing my files in such a way allows me to jump to where I need to in my project faster, without having to scroll through thousands of lines of code.
+
+![Separation of Concerns](https://github.com/i0natan/nodebestpractices/blob/master/assets/images/structurebycomponents.PNG?raw=true)
+
+You could take the Separation of Concerns principle even further by managing these separate modules with independent git repositories, or [creating NPM packages](https://docs.npmjs.com/getting-started/creating-node-modules) of useful code snippets for use in other projects.
+
+---
+## Practice
+
+Which of these functions will import a package into your NodeJS project?
```
Comment (lizTheDeveloper) [inline JavaScript/Node/best-practices/project-struture.md]: This question isn't really about separation of concerns

Comment (milesflo) [reply JavaScript/Node/best-practices/project-struture.md]: I thought it'd be supplementary to the lesson, but I'll do a different one

**Comment by lizTheDeveloper** [comment]
merged on another branch

**Pull request closed without merging**